src/sas/qtgui/Calculators/media/shape_2_sas_help.rst

```diff
@@ -0,0 +1,87 @@
+.. shape_2_sas_help.rst
+
+.. by J Krzywon, NIST, May 2025
+
+.. _Shape2SAS_Tool:
+
+Shape2SAS
+=========
+
+Description
+-----------
+
+This tool allows the user to build particles from a series of predefined shapes. By allowing particles to be placed in
+any location, or orientation relative to an origin, many unique structures can be constructed. Once a structure is
+created, the user can generate a theoretical data set that can be analysed, or an empirical fitting model can be
+generated to compare the structure to real data.
+
+Please note, the particles are constructed from a series of 'atoms' randomly distributed within the defined shape.
+Because of this randomness, creating the same particle two different times may result in subtle differences between
+scattering curves and slight differences in the resulting empirical models.
+
+The base shapes available to the user include spheres, cylinders, ellipsoids, elliptical cylinders, disks, cubes, cuboids,
+hollow spheres, hollow cubes, and cylindrical rings. Each of the particles can be individually modified in all dimensions,
+can be shifted relative to the origin, rotated relative to the particle centroid, and rotated relative to the coordinate
```

Comment:
I am confused (meaning I truly don't understand what is meant or what this does) by this and the explanation below regarding RPX, RPY, and RPZ.  How is the particle rotation relative to the origin different from just shifting the center of mass in x,y, and z and rotating about the particle centroid? and RPX etc are given in units of A not angle? Clearly this needs some more clarity?